gnomad/utils/filtering.py

```diff
@@ -443,3 +443,37 @@ def filter_y_nonpar(
         if isinstance(t, hl.Table)
         else t.filter_rows(non_par_expr)
     )
+
+
+def remove_coverage_outliers(t: Union[hl.MatrixTable, hl.Table]) -> Union[hl.MatrixTable, hl.Table]:
+    """
+    Keep only loci where genome coverage was between 15 and 60
+    """
+    ## make 15 and 60 to parameters
+    criteria = (t.coverage.genomes.mean >= 15) & (t.coverage.genomes.mean <= 60)
+    return t.filter_rows(criteria) if isinstance(t, hl.MatrixTable) else t.filter(criteria)
+
+
+def filter_for_mu(t: Union[hl.MatrixTable, hl.Table]) -> Union[hl.MatrixTable, hl.Table]:
+    """
+    Filter to non-coding annotations, remove GERP outliers
+    GERP cutoffs determined by finding 5% and 95% percentiles on list generated by:
+    ```
+    gerp_data = ht.aggregate(gerp=hl.agg.hist(context_ht.gerp, -12.3, 6.17, 100))
+    cumulative_data = np.cumsum(summary_hist.gerp.bin_freq) + summary_hist.gerp.n_smaller
+    np.append(cumulative_data, [cumulative_data[-1] + summary_hist.gerp.n_larger])
+    list(zip(summary_hist.gerp.bin_edges, cumulative_data / max(cumulative_data)))
+    ```
+    """
+    # This would pull out the consequence specific to each alternate allele, but in the case of intron and intergenic,
```

Comment:
Probably just delete the uncommented code and add comment to docstring

Reply:
i think these comments are all relating to alternative commented out code